src/solvers/unsteady_solver.C

```diff
@@ -306,4 +481,12 @@ Real UnsteadySolver::du(const SystemNorm & norm) const
   return _system.calculate_norm(*solution_copy, norm);
 }
 
+void UnsteadySolver::update()
```

Comment:
In hindsight this was enough of a bugfix that I'd have been happy committing it in its own PR.